src/pages/InkController/SiderComponent/style.scss

```diff
@@ -104,7 +104,7 @@
     color: rgba(243, 243, 243, 0.83);
     height: 48px;
     width: 48px;
-    opacity: 0.5;
+    // opacity: 0.5;
```

Comment:
Let's try with opacity 1.0 for now, but i think we may want to reduce it to at least 0.8 eventually.